client/src/components/player.js

```diff
@@ -16,6 +16,17 @@ function GetRaceIcon(race) {
 }
 
 function Player(player) {
+  let name = <Typography variant="h6">{player.name}</Typography>;
+
+  if(player.aka) {
+    name = (
+      <div>
```

Comment:
I'm new to react, but I think a Fragment would be better here

Reply:
Changed to React.Fragment